packages/modules/create-itwin-viewer/package.json

```diff
@@ -10,14 +10,14 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/iTwin/viewer.git"
+    "url": "git+https://github.com/iTwin/viewer.git"
```

Comment:
what is this change for?

Reply:
it wasn't a manual change, i just reverted it though.